bedrock/firefox/views.py

```diff
@@ -372,6 +372,9 @@ def get_context_data(self, **kwargs):
         # add version to context for use in templates
         ctx['version'] = self.kwargs.get('version') or ''
 
+        # add funnelcake to context for use in templates
+        ctx['funnelcake'] = self.request.GET.get('f', '')
```

Comment:
@pmac I know this is already being done above for the version number by the look of it, but is this a potential XSS vector being inserted into the template? (will Jinja handle this ok, or do we need to try and limit / sanitize the values we pass through)